frontend/src/globals/unity-util.ts

```diff
@@ -674,6 +691,54 @@ export class UnityUtil {
 		}
 	}
 
+	/**
+	 * Called by the Calibration Tool when a user action changes the state of the Vector.
+	 * The line is given as a start position and end position in Project coordinates.
+	 */
+	/** @hidden */
+	public static calibrationVectorChanged(lineJson: string) {
+		const line = JSON.parse(lineJson);
+
+		// eslint-disable-next-line no-console
+		console.log(line);
+
+		if (line.end && line.end.length > 0) {
+			this.calibrationVectorPoints.push(line.start);
+			this.calibrationVectorPoints.push(line.end);
+		}
+	}
+
+	public static calibrationVectorPoints = [];
+
+	public static calibrationVectorTestReset() {
+		this.calibrationVectorPoints = [];
+	}
+
+	public static calibrationVectorTestGetCoords() {
```

Comment:
-  The plane is being cut off if its sticking out of the current model 

![image](https://github.com/3drepo/3drepo.io/assets/42032433/832ca3e1-4ae3-4559-ad61-b6e936c24260)

Reply:
Hi @sanmont3drepo, this should be fixed now.